src/compas_view2/objects/meshobject.py

```diff
@@ -189,12 +180,9 @@ def init(self):
             colors.append(color)
             elements.append([i + 0, i + 1])
             i += 2
-        self._edges = {
-            'positions': make_vertex_buffer(list(flatten(positions))),
-            'colors': make_vertex_buffer(list(flatten(colors))),
-            'elements': make_index_buffer(list(flatten(elements))),
-            'n': i
-        }
+        self._linevertices = positions
+        self._linevertexcolors = colors
+        self._lineelements = elements
```

Comment:
how about
* `self._line_position_vertices`
* `self._line_color_vertices`,
* `self._line_elements`

Reply:
or `attributes` instead of `vertices`?

Reply:
so the the whole set would be :
- self._point_position_attributes
- self._point_color_attributes
- self._point_element_attributes


- self._line_position_attributes
- self._line_color_attributes
- self._line_element_attributes


- self._frontface_position_attributes
- self._frontface_color_attributes
- self._frontface_element_attributes
...

Reply:
Or how about more aligned with the local variable names like:
- self._point_positions
- self._point_colors
- self._point_elements
- self._line_positions
- self._line_colors
- self._line_elements
- ....

Reply:
okay sounds good. in any case, we should differentiate between position/color and element because they are not the same type of buffers...